ui/apps/platform/src/Containers/Audit/ListeningEndpoints/ListeningEndpointsPage.tsx

```diff
@@ -104,6 +104,8 @@ function ListeningEndpointsPage() {
     const entityToggle = useSelectToggle();
     const autocompleteToggle = useSelectToggle();
 
+    const [areAllRowsExpanded, setAllRowsExpanded] = useState<boolean>(false);
```

Comment:
Question for understanding whether `<boolean>` is needed when initial value is `false` or `true`

Reply:
Ah, nope this is extra and unneeded.